app/views/vm_common/_live_migrate.html.haml

```diff
@@ -52,6 +52,23 @@
                   'paging_div_buttons_state_enabled' => true,
                   'paging_div_buttons_id'            => "angular_paging_div_buttons",
                   'paging_div_buttons_type'          => "Submit"}
+- unless @explorer
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
```

Comment:
@mansam you need to add similar code https://github.com/mansam/manageiq/blob/ffea2c9ec18e57814e246b6d39fedaee2bedccfa/app/views/vm_common/_resize.html.haml#L47  to handle enabling/disabling form buttons in live_migrate, evacuate, attach, detach views.